api/db/vars.env

```diff
@@ -12,5 +12,5 @@ MYSQL_ROOT_PASSWORD=root-notasecret
 WORKBENCH_DB_USER=workbench
 WORKBENCH_DB_PASSWORD=wb-notasecret
 # TODO: change to public DB user once it exists
-PUBLIC_DB_USER=workbench
-PUBLIC_DB_PASSWORD=wb-notasecret
+PUBLIC_DB_USER=public
```

Comment:
If you change this now, then everybody needs to set up a public DB locally to run the data browser... are you expecting that change now? (If so, please make sure you tell everybody what to do...)

We also need to update vars.env in GCS to affect the test environment... unless you've already done that?

Reply:
I'm just changed it back. I realize I do not create the public db user yet on the local mysql create. So i will do this in another pr.